src/stan/variational/advi.hpp

```diff
@@ -505,7 +505,9 @@ namespace stan {
                                    tol_rel_obj, max_iterations,
                                    logger, diagnostic_writer);
 
-        // Write mean of posterior approximation on first output line
+        /* Write mean of posterior approximation on first output line.
```

Comment:
Can you revert this comment leaving only the first line.

The second and third lines of the comment should actually be put in `base_family.hpp`, I think. That's where you should explain what the `mean()` function does (or does not do).